tests/test_tag.py

```diff
@@ -52,77 +49,71 @@ def test(self):
         ret = main(["add", self.FOO, self.BAR])
         self.assertEqual(ret, 0)
 
-        with MockLoggerHandlers(logger), ConsoleFontColorsRemover():
-            reset_logger_standard_output()
+        self._caplog.clear()
 
-            ret = main(["tag", "list"])
-            self.assertEqual(ret, 0)
+        ret = main(["tag", "list"])
+        self.assertEqual(ret, 0)
 
-            self.assertEqual("", logger.handlers[0].stream.getvalue())
+        self.assertEqual("", self._caplog.text)
 
         ret = main(["tag", "add", "v1"])
         self.assertEqual(ret, 0)
 
-        with MockLoggerHandlers(logger), ConsoleFontColorsRemover():
-            reset_logger_standard_output()
+        self._caplog.clear()
 
-            ret = main(["tag", "list"])
-            self.assertEqual(ret, 0)
+        ret = main(["tag", "list"])
+        self.assertEqual(ret, 0)
 
-            self.assertEqual(
-                logger.handlers[0].stream.getvalue(),
-                "bar.dvc:\n"
-                "  bar:\n"
-                "    v1:\n"
-                "      md5: 8978c98bb5a48c2fb5f2c4c905768afa\n"
-                "foo.dvc:\n"
-                "  foo:\n"
-                "    v1:\n"
-                "      md5: acbd18db4cc2f85cedef654fccc4a4d8\n"
-                "\n",
-            )
+        expected = (
+            "bar.dvc:\n"
+            "  bar:\n"
+            "    v1:\n"
+            "      md5: 8978c98bb5a48c2fb5f2c4c905768afa\n"
+            "foo.dvc:\n"
+            "  foo:\n"
+            "    v1:\n"
+            "      md5: acbd18db4cc2f85cedef654fccc4a4d8\n"
+        )
+
+        # TODO: Format `caplog` output to capture only "%(message)s"
```

Comment:
This is going to get lost. Need to either fix it, or add an issue for it.

Reply:
As far as I know, there's no way to do this programmatically, @efiop, there's currently an open issue for it, but looks like it will stay in the icebox for a while: https://github.com/pytest-dev/pytest/issues/2987

We could modify the logger format with an option, tho `pytest --log-format="%(message)s"`, but I'm afraid we would be crossing boundaries by coupling logger formatter with just the messages.

Mabye we can monkeypatch caplog to add a `messages` method, but not a fan of that solution.

Reply:
Do we even need it then? If not, let's just remove that TODO.

Reply:
Indeed :+1: